app/controllers/api/v1/application_controller.rb

```diff
@@ -25,9 +25,13 @@ def authentication_error
       render json: { error: I18n.t('errors.401') }, status: 401
     end
 
-    def record_not_found_error
+    def existence_error
```

Comment:
I was trying to go for a simple `<noun>_error` naming convention, but I'm not sure if that makes more or less sense :no_mouth: